subprojects/core/src/main/java/org/gradle/plugin/use/internal/DefaultPluginRequest.java

```diff
@@ -57,6 +57,17 @@ public String getVersion() {
         return version;
     }
 
+    @Nullable
+    @Override
+    public String getArtifact() {
```

Comment:
It should be Object vs String but I don't think it will be resolved from what the user specified by then, unless you want me to call Dependencies.create when someone specifics an artifact.